lib/node_modules/@stdlib/stats/base/dists/chisquare/mgf/package.json

```
@@ -14,11 +14,14 @@
     }
   ],
   "main": "./lib",
+  "gypfile": true,
   "directories": {
     "benchmark": "./benchmark",
     "doc": "./docs",
+    "include": "./include",
     "example": "./examples",
```

Ordered alphabetically to make the CI linter happy: 

```suggestion
    "example": "./examples",
    "include": "./include",
```